src/Utility/CommonOptionsTrait.php

```diff
@@ -52,6 +52,11 @@ protected function _setCommonOptions($parser)
             'help' => 'The theme to use when baking code.',
             'default' => Configure::read('Bake.theme'),
             'choices' => $bakeThemes
+        ])->addOption('view-class', [
+            'short' => 'v',
+            'help' => 'The view class to use baking code.',
+            'default' => Configure::read('Bake.viewClass') ?: 'Bake.Bake',
+            'choices' => ['Bake.Bake', 'Bake.BakeTwig'],
```

Comment:
Why would an end user want to chose the view class?

Reply:
Maybe I have misunderstood how choice is supposed to be used but I thought it's shows the available options to user. One would use `Bake.Bake` to use existing templates or `Bake.BakeTwig` if he/she wants to use twig templates.

Reply:
Could we autodetect if a twig template exists and prefer them over php templates? That would keep the interface to bake simpler and let people incrementally update their custom templates.

Reply:
It would also mean we don't need to keep 2 copies of all the templates around.

Reply:
> Could we autodetect if a twig template exists and prefer them over php templates? That would keep the interface to bake simpler and let people incrementally update their custom templates.

Well a particular task doesn't necessarily use a single template file. For e.g. model task uses multiple templates and elements. So detection of whether twig templates exists won't be practical. I could check for just say existence of the main template `table.twig` and assume all other templates used by the task also exist.

Reply:
Also template existing detection won't be easy when dealing with themes when some templates are used from custom theme and some from the fallback default theme.